ospf6d/ospf6_abr.c

```diff
@@ -853,8 +858,9 @@ void ospf6_abr_examin_summary(struct ospf6_lsa *lsa, struct ospf6_area *oa)
 	if (lsa->header->type == htons(OSPF6_LSTYPE_INTER_PREFIX)) {
 		if (IS_OSPF6_DEBUG_EXAMIN(INTER_PREFIX)) {
 			is_debug++;
-			zlog_debug("%s: Examin %s in area %s", __func__,
-				   lsa->name, oa->name);
+			zlog_debug("%s: Examin %s age %d in area %s", __func__,
```

Comment:
any chance we could fix the not-typo-but-really-a-typo while we're in here?

Reply:
resolved